src/main/java/com/stripe/model/Balance.java

```diff
@@ -15,51 +17,100 @@
-  String object;
+  /**
+   * Funds that are available to be transferred or paid out, whether automatically by Stripe or
+   * explicitly via the [Transfers API](#transfers) or [Payouts API](#payouts). The available
+   * balance for each currency and payment type can be found in the `source_types` property.
+   */
+  @SerializedName("available")
   List<Money> available;
+
+  /**
+   * Funds held due to negative balances on connected Custom accounts. The connect reserve balance
+   * for each currency and payment type can be found in the `source_types` property.
+   */
+  @SerializedName("connect_reserved")
+  List<Money> connectReserved;
+
+  /**
+   * Has the value `true` if the object exists in live mode or the value `false` if the object
+   * exists in test mode.
+   */
+  @SerializedName("livemode")
   Boolean livemode;
+
+  /** String representing the object's type. Objects of the same type share the same value. */
+  @SerializedName("object")
+  String object;
+
+  /**
+   * Funds that are not yet available in the balance, due to the 7-day rolling pay cycle. The
+   * pending balance for each currency, and for each payment type, can be found in the
+   * `source_types` property.
+   */
+  @SerializedName("pending")
   List<Money> pending;
 
-  // <editor-fold desc="retrieve">
   /**
-   * Retrieve balance.
+   * Retrieves the current account balance, based on the authentication that was used to make the
+   * request. For a sample request, see <a
+   * href="/docs/connect/account-balances#accounting-for-negative-balances">Accounting for negative
+   * balances</a>.
    */
   public static Balance retrieve() throws StripeException {
-    return retrieve((RequestOptions) null);
+    return retrieve((Map<String, Object>) null, (RequestOptions) null);
   }
 
   /**
-   * Retrieve balance.
+   * Retrieves the current account balance, based on the authentication that was used to make the
+   * request. For a sample request, see <a
+   * href="/docs/connect/account-balances#accounting-for-negative-balances">Accounting for negative
+   * balances</a>.
    */
   public static Balance retrieve(RequestOptions options) throws StripeException {
-    return retrieve(null, options);
+    return retrieve((Map<String, Object>) null, options);
   }
 
   /**
-   * Retrieve balance.
+   * Retrieves the current account balance, based on the authentication that was used to make the
+   * request. For a sample request, see <a
+   * href="/docs/connect/account-balances#accounting-for-negative-balances">Accounting for negative
+   * balances</a>.
    */
   public static Balance retrieve(Map<String, Object> params, RequestOptions options)
       throws StripeException {
-    return request(RequestMethod.GET, singleClassUrl(Balance.class), params, Balance.class,
-        options);
+    String url = String.format("%s%s", Stripe.getApiBase(), "/v1/balance");
+    return request(ApiResource.RequestMethod.GET, url, params, Balance.class, options);
```

Comment:
4) Retrieve (standard): Static method to retrieve a singleton-like resource, taking no ID.

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/Balance.java#L80-L84